packages/shared/styles/components/atoms/SfProperty.scss

```diff
@@ -2,45 +2,38 @@
 
 .sf-property {
   display: flex;
-  align-items: center;
   color: var(--property-color, var(--c-text));
   &__name {
-    display: flex;
-    align-items: center;
-    margin: var(--property-name-margin, 0 var(--spacer) 0 0);
-    color: var(--property-name-color, var(--c-text-muted));
-    font: var(
-      --property-name-font,
-      font(
-        --property-name,
-        400,
-        var(--font-size-regular),
-        1.6,
-        var(--body-font-family-secondary)
-      )
-    );
+    margin: var(--property-name-margin, 0 var(--spacer-extra-small) 0 0);
+    color: var(--property-name-color, var(--c-secondary-variant));
     text-transform: var(--property-name-text-transform);
     &::after {
       content: var(--property-name-content, ":");
     }
   }
-  &__value {
-    font: var(
-      --property-value-font,
-      font(
-        --property-value-font,
-        400,
-        var(--font-size-regular),
-        1.6,
-        var(--body-font-family-secondary)
-      )
-    );
-  }
+  @include font(
+    --property-font,
+    var(--font-weight-regular),
+    var(--font-size-extra-small),
+    1.2,
+    var(--body-font-family-secondary)
+  );
   &--full-width {
-    --property-name-text-transform: uppercase;
     justify-content: space-between;
+    width: 100%;
+  }
+  &--large {
```

Comment:
for --large also the value `font-weight` should be bold or semibold (ask Marcin, as we can't check it on designs)

Reply:
@aniamusial you can copy the project to your Figma and check it :)

Reply:
value is really semibold as you mentioned. I'll changed that.

Reply:
I pushed this change, but font-wieght is set by font-weight css property not custom property because that doesn't work for me.

Reply:
Font variables updated.